### DISCUSSION TITLE
Tips running on read replicas

### DISCUSSION THREAD

**Discussion by tranpeter** [microsoft/azure-sql-tips#71] [General]
Thank your for this script! It's great but my issue is a lot of my issue is on the read replicas. Can we create a modified version that can be used against the read replica database?

**Comment by dimitri-furman** [comment]
The script supports read replicas. See [Usage suggestions](https://github.com/microsoft/azure-sql-tips/wiki/Azure-SQL-Database-tips#usage-suggestions).
